req.py

```diff
@@ -171,29 +171,31 @@ def fill_out_application_and_submit(driver, rand_num):
     print(f"successfully submitted application")
 
 def main():
-    rand_num = random.randint(0, 3)
     i = 1
     while (i < 10000):
-
+        rand_num = random.randint(0, 3)
         try:
             driver = start_driver(rand_num)
         except Exception as e:
             print(f"failed to start driver: {str(e)}")
-            pass
+            driver.close()
+            continue
 
         time.sleep(2)
 
         try:
             generate_account(driver, rand_num)
         except Exception as e:
             print(f"failed to create account: {str(e)}")
-            pass
+            driver.close()
+            continue
 
         try:
             fill_out_application_and_submit(driver, rand_num)
         except Exception as e:
             print(f"failed to fill out app and submit: {str(e)}")
-            pass
+            driver.close()
+            continue
 
         driver.close()

```

Comment:
i'm surprised this doesn't break when one of the exceptions hit the continue statement bc then don't we call `driver.close()` twice?